Mathlib/Combinatorics/Graph/Basic.lean

```diff
@@ -16,37 +16,37 @@ and there may also be more than one edge `e` whose ends are the same pair `(x,y)
 A multigraph where neither of these occurs is 'simple',
 and these objects are described by `SimpleGraph`.
 
-This module defines `Graph α β` for a vertex type `α` and an edge type `β`,
+This module defines `Graph α ε` for a vertex type `α` and an edge type `ε`,
```

Comment:
I think keeping beta here is quite a bit nicer than epsilon!

Reply:
I don't have the same feeling about relative ugliness of greek letters. My rationale was that epsilon suggests E for edge, and that beta looks weird if there are vertices called `a, b, c`. 

But I can change it back if you prefer.